### ISSUE TITLE
Screen reader with keyboard shortcuts

### ISSUE THREAD

**Issue by clapierre** [jupyterlab/jupyterlab#6573]
@ivanov commented on May 15
Using NVDA - we can confirm that

Ctrl-Shift-] and Ctrl-Shift-[ work for switching tab navigation, but no announcement of the tab name is made - so the user has no idea what tab they switched to.

Ctrl-B works to toggle the left pane, but again, does not announce.

Ctrl-Shift-B to toggle the single document mode works, but again does not announce.

Ctrl-Shift-C to focus the command palette works, but again does not announce.

Tab completion behavior - work, not announcing when initially invoked, and improperly repeating the text already typed, and not announcing the options themselves as they are navigated through.

Ctrl-S works to save, but does not announce the status. It should use an aria-live region.

Additionally, after discussing with James and Neil - you might want to enable that behavior to be customizable, to allow screen reader users to opt in and out of this behavior.

Ctrl-Shift-S brings up a non-real-dialog, but Tab behavior does allow navigation between those buttons.

Ctrl-F and Ctrl-G work - but no announcement of navigation. The default browser behavior announces the number of the result, at least.

Ctrl-Shift-L works - but again, no announcement (feedback) about a new panel having been created.

**Comment by tonyfast** [comment]
these descriptions relate the lack of status messages referenced in #9399
